<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(245, 245, 225)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1080</width>
      <height>720</height>
     </rect>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>551</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="header">
       <item>
        <widget class="QLabel" name="work">
         <property name="font">
          <font>
           <family>함초롬바탕</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>업무</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="announcement">
         <property name="font">
          <font>
           <family>함초롬바탕</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>공지사항</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="management">
         <property name="font">
          <font>
           <family>함초롬바탕</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>관리</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::AutoText</enum>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="work_line">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>118</width>
        <height>5</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame {
	color: rgb(221, 160, 221);
	background-color: rgb(221, 160, 221);
    max-height: 3px;
    min-height: 3px;
}
</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="midLineWidth">
       <number>6</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>101</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>160</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>고객이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="work_frame">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>110</y>
        <width>120</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#work_frame {
    border-style: solid;
    border-color: rgb(221, 160, 221); 
    border-width: 2px;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>고객 조회</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>신규 계좌 개설</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>235</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #fdfdea;
    color: #4a4a4a;
    border: 2px solid #dda0dd;
    border-radius: 12px;
    padding: 6px 14px;
}
QPushButton:hover {
    background-color: #e6cce6;
}
QPushButton:pressed {
    background-color: #c183c1;
    border-style: inset;
}

</string>
      </property>
      <property name="text">
       <string>검색</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>361</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget {
alternate-background-color: rgb(250, 250, 240);
    background-color: rgb(245, 245, 225);     /* 아이보리 배경 */
    color: #3a3a3a;                            /* 진한 회색 텍스트 */
    gridline-color: rgb(221, 160, 221);        /* 플럼 색상의 셀 경계선 */
    font-size: 14px;
    selection-background-color: rgb(221, 160, 221); /* 선택 시 배경 */
    selection-color: white;                   /* 선택 시 텍스트 */
    border: none;
}

QHeaderView::section {
    background-color: rgb(221, 160, 221);     /* 플럼 헤더 */
    color: white;
    padding: 4px;
    border: none;
    font-weight: bold;
}

QTableWidget::item {
    padding: 6px;
    border: none;
}

QTableWidget::item:selected {
    background-color: rgb(221, 160, 221);     /* 선택 셀 배경 */
    color: white;
}

</string>
      </property>
      <row/>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>계좌</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>생년월일</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
